externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/PlanOnlyStatement.scala

```diff
@@ -32,7 +31,8 @@ import org.apache.kyuubi.session.Session
 class PlanOnlyStatement(
     session: Session,
     override val statement: String,
-    mode: OperationMode)
+    mode: OperationMode,
+    skip: Seq[String])
```

Comment:
we can move it out of the class parameters to make it simple，like

private val exculdedPlans = spark.conf.getOrElse(xxx, session.getConf.get)